_source/_posts/2023-01-23-keycloak-kubernetes-prod.md

```diff
@@ -0,0 +1,637 @@
+---
+layout: blog_post
+title: "OAuth2 microservices with Keycloak on Google Kubernetes Engine"
+author:
+by: contractor
+communities: [devops,security,java]
+description: "A walk-through of building a microservices architecture with JHipster, Keycloak for OAuth2, and Google Kubernetes Engine (GKE) deployment. Keycloak as identity broker with Auth0 as the identity provider."
```

Comment:
```suggestion
description: "A walk-through of building a microservices architecture with JHipster, Keycloak for OAuth 2.0, and Google Kubernetes Engine (GKE) deployment."
```

Descriptions should be around 160 characters for SEO. https://moz.com/learn/seo/meta-description